feat(a380x): OANS advanced guide: Add fallback mode documentation (#1032)
flybywiresim/docs: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md

![No selection](../assets/advanced-guides/oans-btv/btv-setup-no-selection.png)
![Runway + exit selected](../assets/advanced-guides/oans-btv/btv-setup-selected.png)

> To finalize the setup, the auto brake mode selector knob is turned to the "BTV" option, which arms BTV (this should be

Reviewer: The flow for the image here seems a bit weird. I have a couple of suggestions.

1. Add a header titled `Finalize BTV Setup` in an H3. And link to that to separate it as finalizing BTV IMO is concurrent with both having navigraph or not.
2. Wrap the without navigraph unlimited in a `???` admonition to collapse it and users can open that at will if they don't have it.

Suggestions will be provided below via review.

> checked in the PFD FMA). Images [see below](#without-navigraph-unlimited).

Reviewer: ```suggestion
checked in the PFD FMA). Images [see below](#finalize-btv-setup).
```
in line with option 1 above

> ### Without Navigraph Unlimited
> 
> Provided the landing runway is set in the FMS, BTV can also be set up without the airport map from the OANS. Turn the EFIS range
> selector to ZOOM, and open the OANS control panel by clicking on the arrow symbol on the lower right of the ND.
> 
> Change to the "MAP DATA" tab, check that the selected landing runway is correct and enter the desired stopping distance in the input
> field next to "BTV STOP DISTANCE" (don't forget to finish by pressing the enter key).
> 
> ![BTV STOP DISTANCE selection in OANS control panel](../assets/advanced-guides/oans-btv/btv-fallback.png)

Reviewer: ```suggestion
??? info "Without Navigraph Unlimited"
	### Without Navigraph Unlimited
	
	Provided the landing runway is set in the FMS, BTV can also be set up without the airport map from the OANS. Turn the EFIS range
	selector to ZOOM, and open the OANS control panel by clicking on the arrow symbol on the lower right of the ND.
	
	Change to the "MAP DATA" tab, check that the selected landing runway is correct and enter the desired stopping distance in the input
	field next to "BTV STOP DISTANCE" (don't forget to finish by pressing the enter key).
	
	![BTV STOP DISTANCE selection in OANS control panel](../assets/advanced-guides/oans-btv/btv-fallback.png)
```

In line with option 2.

Must keep h3 inside to have it in the TOC and for the rest of your internal links.

> 

Reviewer: ```suggestion
### Finalize BTV Setup
```

In line with my option 1 above.
